File: Module.Persist.TPM/Model/History/HistoricalCompetitorBrandTech.cs
using System;
using Core.History;
using Module.Persist.TPM.Model.TPM;

namespace Module.Persist.TPM.Model.History
{
    [AssociatedWith(typeof(CompetitorBrandTech))]
    public class HistoricalCompetitorBrandTech : BaseHistoricalEntity<Guid>
    {
        public string BrandTech { get; set; }
        public string Color { get; set; }
        public string CompetitorName { get; set; }
    }
}

File: Module.Frontend.TPM/Client/model/tpm/competitorbrandtech/HistoricalCompetitorBrandTech.js
Ext.define('App.model.tpm.competitorbrandtech.HistoricalCompetitorBrandTech', {
    extend: 'Ext.data.Model',
    idProperty: '_Id',
    breezeEntityType: 'CompetitorBrandTech',
    fields: [
        { name: '_Id', type: 'string', hidden: true },
        { name: '_ObjectId', hidden: true },
        { name: '_User', type: 'string', isDefault: true },
        { name: '_Role', type: 'string', isDefault: true },
        { name: '_EditDate', type: 'date', isDefault: true, timeZone: +3, convert: dateConvertTimeZone },
        { name: '_Operation', type: 'string', isDefault: true },
        { name: 'Color', type: 'string', isDefault: true },
        { name: 'CompetitorName', type: 'string', isDefault: true },
        { name: 'BrandTech', type: 'string', isDefault: true }
    ],
    proxy: {
        type: 'breeze',
        resourceName: 'HistoricalCompetitorBrandTechs',
        reader: {
            type: 'json',
            totalProperty: 'inlineCount',
            root: 'results'
        },
        extraParams: {
            //Id промо для метода GetById в истории
            Id: null
        }
    }
});
